android/src/main/java/com/swmansion/reanimated/Utils.java

```diff
@@ -35,4 +35,15 @@ public static String simplifyStringNumbersList(String list) {
     // to make usage of std::istringstream in C++ easier
     return list.replace(",", "").replace("[", "").replace("]", "");
   }
+
+  public static float convertToFloat(Object value) {
+    if (value instanceof Integer) {
+      return ((Integer) value).floatValue();
+    } else if (value instanceof Float) {
+      return (float) value;
```

Comment:
Why is this case not consistent with the rest? Shouldn't it be this way?
```suggestion
      return ((Float) value).floatValue();
```

Reply:
IDE suggest to avoid it:
![image](https://github.com/software-mansion/react-native-reanimated/assets/36106620/a2b099fb-b5e3-4e08-ae52-937a161ec2b6)